src/mote/main.py

```diff
@@ -123,6 +130,42 @@ def mainpage():
     return render_template("mainpage.html")
 
 
+def fetch_recently_completed_meetings():
+    global is_running
+    while is_running:
+        datagrepper_base_url = "https://apps.fedoraproject.org"
+        topic = "org.fedoraproject.prod.meetbot.meeting.complete"
+        call_period = 60
+        source = "{}/datagrepper/raw?delta={}&topic={}".format(
+            datagrepper_base_url, call_period, topic
+        )
+        parse_object = json.loads(ulrq.urlopen(source).read().decode())
+        meeting_count = parse_object["total"]
+        if meeting_count != 0:
+            meeting_rawlist = parse_object["raw_messages"]
+            socketio.emit("show_toast", meeting_rawlist)
+        time.sleep(60)
+
+
+@socketio.on("connect")
+def trigger_on_connect():
+    global client_count
+    global is_running
+    client_count += 1
+    if is_running == False:
+        is_running = True
+        fetch_recently_completed_meetings()
+
+
+@socketio.on("disconnect")
+def trigger_on_disconnect():
+    global client_count
+    global is_running
+    client_count -= 1
+    if client_count == 0:
+        is_running = False
```

Comment:
When the last client disconnects, there is a 60 seconds period when `is_running` is False, but the while loop is still running the sleep call.
If another client connects during this time, it starts a new loop, and the previous one keeps running.
You need to make sure the loop is stopped before running a new one.